packages/relay-runtime/store/observeFragmentExperimental.js

```diff
@@ -173,6 +170,67 @@ function observeSelector<TFragmentType: FragmentType, TData>(
   });
 }
 
+function observePluralSelector<
+  TFragmentType: FragmentType,
+  TData: Array<mixed>,
+>(
+  environment: IEnvironment,
+  fragmentNode: Fragment<TFragmentType, TData>,
+  fragmentSelector: PluralReaderSelector,
+): Observable<FragmentState<TData>> {
+  const snapshots = fragmentSelector.selectors.map(selector =>
+    environment.lookup(selector),
+  );
+
+  return Observable.create(sink => {
+    const states = snapshots.map((snapshot, index) =>
+      snapshotToFragmentState(
+        environment,
+        fragmentNode,
+        fragmentSelector.selectors[0].owner,
```

Comment:
Either use the `index` here or omit it from the callback arguments on line 186

Reply:
https://github.com/facebook/relay/pull/4862/commits/70f047e6725e7aa9e60277e6b15054b7405e07ee